ci/run_ci.sh

```diff
@@ -26,6 +26,9 @@ Workload                | Test                           | Result | Runtime  |
 ------------------------|--------------------------------|--------|----------|
 EOF
 
+# Clear the /tmp/ripsaw* directory to avoid pip version conflicts due to existing temp files.
+rm -rf /tmp/ripsaw*
```

Comment:
2 Comments:

1) Since the install_cli function in utils/benchmark-operator.sh hard codes the path to /tmp/ripsaw-cli we should use the full path name and not wildcard.

2) Since we run the install_cli function EACH test we should probably move the rm to the top of the for loop below so we clear it each time.

Reply:
@dry923 Fixed :+1: